### PR TITLE
Add search bar and sorting

### LINKED ISSUE #33: Add a search to home
Check out fuzzy string matching for now

### PR DESCRIPTION
**Pull request by phillip-niakoulov**

### CODE REVIEW

File: backend/routes/productRoutes.js [hunk 1]
```diff
@@ -104,15 +105,58 @@ router.post(
 );
 
 // Get all products
+// UNUSED!
 router.get('/', async (req, res) => {
```
Comment (curtisjbradley) [inline backend/routes/productRoutes.js:109]: Looks good.

Let's remove this unused code.

File: frontend/src/components/ProductList.jsx
Comment (heyitsjorge) [inline frontend/src/components/ProductList.jsx:1]: Great way to clean up Home.jsx and use ProductList.jsx for the visibility of product functionality!

**Review by heyitsjorge — COMMENTED** [review]
Good Stuff Phillip!

File: frontend/src/components/ProductList.jsx
Comment (edsonmunoz) [reply frontend/src/components/ProductList.jsx:1]: Good Work, looks clean!!!
Way to simplify the home page and have all the logic in the Product List

**Pull request merged**